test/features/managet_test_spider.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Features.Manager.TestSpider do
```

Comment:
typo in a file name.

Reply:
Isn't it possible to re-use one of already defined test spiders?

Reply:
this test_spider was defined inside the ManagerTest. I'm splitting it out to a commonplace so others tests can use it